packages/flutter_tools/lib/src/ios/ios_emulators.dart

```diff
@@ -39,39 +48,16 @@ class IOSEmulator extends Emulator {
 
   @override
   Future<void> launch() async {
-    Future<bool> launchSimulator(List<String> additionalArgs) async {
-      final List<String> args = <String>[
-        'open',
-        ...additionalArgs,
-        '-a',
-        globals.xcode.getSimulatorPath(),
-      ];
-
-      final RunResult launchResult = await globals.processUtils.run(args);
-      if (launchResult.exitCode != 0) {
-        globals.printError('$launchResult');
-        return false;
-      }
-      return true;
-    }
-
-    // First run with `-n` to force a device to boot if there isn't already one
-    if (!await launchSimulator(<String>['-n'])) {
-      return;
+    final RunResult launchResult = await globals.processUtils.run(<String>[
+      'open',
+      '-a',
+      globals.xcode.getSimulatorPath(),
+    ]);
+    if (launchResult.exitCode != 0) {
+      globals.printError('$launchResult');
+      return false;
     }
 
-    // Run again to force it to Foreground (using -n doesn't force existing
-    // devices to the foreground)
```

Comment:
We don't need to run `open` a second time since the next thing that happens is booting the specified simulator.